### DISCUSSION TITLE
How do I append certain images in a folder?

### DISCUSSION THREAD

**Discussion by marfmamow** [ImageMagick/ImageMagick#1928] [General]
I am trying to merge 40 images into 4 10 images long jpgs. I am currently copying 10 images, pasting into a new folder, running the command 'convert -append in-*.jpg out.jpg'. I want to know if there's a way to keep all the 40 images in the folder, but only append the 10 images at a time.

I tried using the 'convert in-1.jpg in-5.jpg in-N.jpg +append out-in1-plus-in5-and-inN.jpg' command (with the numbers changed to fit my images) but it was annoying having to type in-1 to in-10, in-11 to in-20, etc. Is there an easy way to merge the images vertically 10 at a time? The images are also in a certain order, so they can't be merged randomly

**Comment by snibgo** [comment]
Instead of a list of input files, you can use "in-%d.jpg[1-10]" and "in-%d.jpg[11-20]" and so on.
